### PR TITLE
Update install-prod-dockerless.md

### PR DESCRIPTION
**Pull request by marcusyoung**
Just a few minor amendments based on my experience of doing a dockerless install

### CODE REVIEW

**Comment by mouse-reeve** [comment]
@CactiChameleon9 would you be down to review these changes?

**Comment by CactiChameleon9** [comment]
Happy to review, not sure how too though... never done that before xD. I assume I need permission?

**Comment by mouse-reeve** [comment]
Nothing formal, just take a look and see if it seems good to you and comment back here. I couldn't figure out how to get github to do its built-in review process

File: content/running_bookwyrm/install-prod-dockerless.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ GRANT ALL PRIVILEGES ON DATABASE bookwyrm TO bookwyrm;
     - Make the system bookwyrm user:
         `useradd bookwyrm -r`
     - Change the owner of your install directory to bookwyrm:
-        `chown -R bookwyrm:bookwyrm /opt/bookwyrm`
+        `chown -R /opt/bookwyrm bookwyrm:bookwyrm`
```
Comment (codeurimpulsif) [inline content/running_bookwyrm/install-prod-dockerless.md:85]: I think this is a mistake, the `chown` command work like `chown -R user:group directory`.

**Comment by codeurimpulsif** [comment]
But all other modifications seems good, I can confirm packages `python3-pip` and `python3-dev` are needed to install dependencies (`python3-dev` is needed to have `python.h` to buid the `rcssmin` dependency).

File: content/running_bookwyrm/install-prod-dockerless.md (hunk 1 above)
Comment (marcusyoung) [reply content/running_bookwyrm/install-prod-dockerless.md]: corrected.

**Comment by CactiChameleon9** [comment]
@marcusyoung I have a few questions.

> You will need to set up a Cron job for the service to start automatically on a server restart.
Isn't that the WHOLE point of the systemd service above, can you explain the point of a cron job clearer (here to me and in the doc) and maybe add in an example cron job.

> python3-pip
Is this needed even for inside the venv? I assumed that pip was packaged by default inside venv - could be wrong. I will need to check if you are unsure.

> python3-dev
Is this needed for Pip? Can you explain (here) why you added this.

The commits should also probably be squashed before merging if they do the same thing ;)

**Comment by marcusyoung** [comment]
@CactiChameleon9 
> Isn't that the WHOLE point of the systemd service above, can you explain the point of a cron job clearer (here to me and in the doc) and maybe add in an example cron job.
> 
The step in the existing document configures Bookwyrm to run as a service. This allows you to use systemctl commands to start and stop the service. However, it does not start the service automatically when the system itself starts/restarts.

> > python3-pip
> 
> Is this needed even for inside the venv? I assumed that pip was packaged by default inside venv - could be wrong. I will need to check if you are unsure.
> 
Pip was needed to be installed on the system outside of venv. I forget at what point, but it was. As confirmed by @codeurimpulsif above.

> The commits should also probably be squashed before merging if they do the same thing ;)
Well, that is not preferred by all maintainers. I'm just doing what I usually do for projects I contribute to.

**Comment by CactiChameleon9** [comment]
Cool, no worries then. Thanks for your detailed response and patience (sorry if I came off as a bit annoyed in my writing 😅)

I think removing the cron mention from the PR makes sense, not up to me though - systemd enable is implied with a more complex install and is more preference than (set in) stone.

I will leave the rest of the PR process to the maintainers (handing back to you @mouse-reeve)

**Comment by mouse-reeve** [comment]
Thank you both! I'm falling on the side of not including the cron bit, since it sounds like there's an element of personal preference in how the admin approaches getting services up and running, and alternatives with systemd.

**Comment by CactiChameleon9** [comment]
@mouse-reeve Sorry to bother, but why was this merged when you were unsure on the cron bit and there had been no change or comment on that?

**Comment by mouse-reeve** [comment]
Because it didn't seem like a big thing and I didn't want to PR to get out of date and become hard to merge. I'm happy to change that part it if you prefer

**Comment by marcusyoung** [comment]
Sorry, that's probably my fault for not following up. Just snowed under at the moment.

**Comment by mouse-reeve** [comment]
No worries at all, I appreciate all your work Marcus!

**Pull request merged**